### ISSUE TITLE
Error while installing - conflict

### ISSUE THREAD

**Issue by masiorama** [aelvan/mailchimp-subscribe-craft#51]
I cannot install the plugin via shell using `composer require aelvan/mailchimp-subscribe`:

```
Using version ^2.0 for aelvan/mailchimp-subscribe
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for aelvan/mailchimp-subscribe ^2.0 -> satisfiable by aelvan/mailchimp-subscribe[2.0.0].
    - aelvan/mailchimp-subscribe 2.0.0 requires pacely/mailchimp-apiv3 dev-master -> satisfiable by pacely/mailchimp-apiv3[dev-master] but these conflict with your requirements or minimum-stability.
Installation failed, reverting ./composer.json to its original content.
```

I'm on Craft 3 RC5.
What am I doing wrong? Thanks in advance.

**Comment by sjenset** [comment]
You are - per say - not doing anything wrong; the package simply depends upon a sub-package which doesn't mean your projects minimum requirements.

Either wait for the mailchimp-apiv3 package to reach release-status, or tweak your minimum-stability to accept dev-packages (nb: This will mean that all other packages you've installed can also be installed as dev-packages which might have unforeseen consequences)

If you want to go for the dev-package approach, simply add the following to your composer.json

`"minimum-stability": "dev",`

**Comment by masiorama** [comment]
Thanks for the clarification!

**Comment by sjenset** [comment]
Actually, what I would do (read: what I just did) to solve this is the following:

`composer require pacely/mailchimp-apiv3 @dev`
`composer require aelvan/mailchimp-subscribe`

That way - only the `pacely/mailchimp-apiv3`-package is installed below my regular stability-requirements and I still get to enjoy this package :)

**Comment by janhenckens** [comment]
I'll jump in on this issue as I just had the same issue. 

I see the dependency has released versions, so you could require those? https://github.com/pacely/mailchimp-api-v3/releases

**Comment by aelvan** [comment]
Fixed in 2.0.1.
